I.write-tests/05-get-second-third-smallest/get-second-third.test.js

```diff
@@ -1,5 +1,13 @@
-// example
-// input = [90, 5, 11, 8, 6];
+
+
+test('get the second and third smallest',function() {
+    var getSecondAndThird = require('./get-second-third');
+    var numbers = [90, 5, 11, 8, 6];
+    var result = getSecondAndThird(numbers);
+    expect(result).toEqual([6,8]); 
+    //expect(numbers).toEqual([90,5,11,8,6]);
```

Comment:
This line is actually correct, but unfortunately the code that was provided for you had a bug in it so wasn't working correctly. Sorry about that 😞 I'll make sure it's fixed for next time.